govc/appliance/logging/forwarding.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	vlogging "github.com/vmware/govmomi/vapi/appliance/logging"
+)
+
+func main() {
+	fmt.Println("vim-go")
+}
+
+type forwarding struct {
+	*flags.ClientFlag
+	*flags.OutputFlag
+}
+
+func init() {
+	cli.Register("appliance.logging.forwarding", &forwarding{})
```

Comment:
Should we want to add support for `set` or `test`, how about naming this `appliance.logging.forwarding.info` ?
Also wondering if we should shorten `appliance` to `vcsa` or even `vcenter`. And `logging` to `log` ?
`vcsa.log.forwarding.info` ?

Reply:
I like `vcsa.log.forwarding.info`.  

Will try adding a setter.  Standby.

Reply:
No need to add a setter until it is needed, I was just looking at what else the API can do.